source/Calamari/Kubernetes/ResourceStatus/Options.cs

```diff
@@ -0,0 +1,7 @@
+namespace Calamari.Kubernetes.ResourceStatus
+{
+    public class Options
```

Comment:
A more descriptive name like `ResourceStatusOptions` will make it easier to follow what this is used for.

Reply:
I didn't use a long name because it is already in the fully qualified name of this class `Calamari.Kubernetes.ResourceStatus.Options`. If the short name will result in a clash with another `Option` class, people can use it as `using ResourceStatusOptions = Calamari.Kubernetes.ResourceStatus.Options`.

Maybe adding some XMLDoc would be helpful here?

Reply:
Added docs for this one.